src/parser/ast.rs

```diff
@@ -95,6 +95,7 @@ pub struct VariableDeclaration {
 }
 
 #[derive(Debug, PartialEq)]
+#[allow(clippy::vec_box)]
```

Comment:
Actually I think clippy is right here. This should be fixed (and I think I already fixed this)